backend/api/src/get-mod-reports.ts

```diff
@@ -0,0 +1,26 @@
+import { ModReport } from 'common/mod-report'
+import { type APIHandler } from './helpers/endpoint'
+import { createSupabaseDirectClient } from 'shared/supabase/init'
+
+export const getModReports: APIHandler<'get-mod-reports'> = async () => {
+  const pg = createSupabaseDirectClient()
+
+  const reports = await pg.many<ModReport>(`
+    SELECT 
+      mr.*, 
+      cc.data->>'content' AS comment_content, 
+      c.question AS contract_question, 
+      c.slug AS contract_slug, 
+      creator.username AS creator_username,
+      owner.data->>'username' AS owner_username,
+      owner.data->>'avatarUrl' AS owner_avatarUrl, 
+      (owner.data->>'isBannedFromPosting')::boolean AS owner_isBannedFromPosting
+    FROM mod_reports mr
```

Comment:
Don't use capital letters in sql. It will come out as lowercase. I think that might be your bug

Reply:
I.e. `owner_avatarUrl => owner_avatar_url`
`owner_isBannedFromPosting => owner_is_banned_from_posting`